core/src/main/scala/doric/syntax/ColGetters.scala

```diff
@@ -188,8 +190,8 @@ private[doric] trait ColGetters[F[_]] {
     * @return
     *   the struct column reference.
     */
-  def colStruct(colName: CName)(implicit location: Location): F[DStruct] =
-    col[DStruct](colName)
+  def colStruct(colName: CName)(implicit location: Location): F[Row] =
```

Comment:
I do not understand this

Reply:
Now every elemt that can be a sparktype of doric must be something matrialiazable. Dstruct can't hold elements, and spark already provide us with row